packages/rich-text/src/helpers/compose.ts

```diff
@@ -0,0 +1,6 @@
+export function compose(...fns: Function[]) {
```

Comment:
Should we get this from some library?

Reply:
You could use https://docs-lodash.com/v4/flow/